hype-gcs/src/main/java/com/spotify/hype/gcs/ManifestUtil.java

```diff
@@ -0,0 +1,81 @@
+/*-
+ * -\-\-
+ * hype-gcs
+ * --
+ * Copyright (C) 2016 - 2017 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.hype.gcs;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class ManifestUtil {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ManifestUtil.class);
+
+  private static final char LAMBDA = 'l';
```

Comment:
Why do you need this? Wouldn't the capsule know where the continuation file is, and everything that was uploaded to GCS should just be in classpath of the continuation (kinda the DF model)? Or is that polluting classpath?